src/main/java/uk/gov/hmcts/reform/sscs/messaging/HearingUpdate.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.sscs.messaging;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class HearingUpdate {
+
+    private String hearingResponseReceivedDateTime;
```

Comment:
Could you please double check the types? Some of those should be LocalDates and enums

Reply:
This class needs to be put into the model package